content/applications/marketing/events/sell_tickets.rst

```diff
@@ -1,68 +1,137 @@
+Lines` tab. The user can proceed to confirm and close the sale, per the usual process.
 
-.. image:: sell_tickets/events-sales-smartbutton.png
+Sell event tickets through the Website app
+==========================================
+
+When a visitor arrives on the register page of the event website, they can click the
+:guilabel:`Register` button to purchase a ticket to the event.
+
+.. note::
+   If the visitor is *not* already on the register page of the event website, clicking
+   :guilabel:`Register` on the event website's submenu redirects them to the proper
+   :guilabel:`Register` page. From there, they can click the :guilabel:`Register` button to begin
+   the ticket purchasing process.
+
+If different ticket tiers are configured for the event, the visitor is presented with a
+:guilabel:`Tickets` pop-up window.
+
+.. image:: sell_tickets/tickets-popup.png
    :align: center
-   :alt: View of an event's form and the sales smart button in Odoo Events.
+   :alt: The tickets pop-up window that appears on the event's website when 'Register' is clicked.
+
+From here, visitors select which ticket tier they would like to purchase, along with a quantity,
+using the numerical drop-down menu available to the right of their desired ticket. Once the desired
+selections have been entered, the visitor then clicks the :guilabel:`Register` button.
 
-.. image:: sell_tickets/events-tickets-registration-product.png
+Then, an :guilabel:`Attendees` pop-up window appears, containing all the questions that have been
+configured in the *Questions* tab of the event form for this particular event.
+
+.. image:: sell_tickets/attendees-popup.png
    :align: center
-   :alt: View of an event form highlighting the column product under the tickets tab in Odoo
-         Events.
+   :alt: The attendees pop-up window that appears on the event's website when 'Ok' is clicked.
 
-Sell tickets through the website
-================================
+If multiple tickets are being purchase at once, there are numbererd sections for each individual
+ticket registrant, each containing the same questions. However, if any question has been configured
+with the *Ask once per order* setting, that question is only asked once -- and **not** for every
+attendee making the reservation in the order.
 
-With tickets purchased through the website, the process is similar to creating a :guilabel:`Sales
-Order` with a specific :guilabel:`Registration` product. Here, tickets are added to a virtual cart,
-and the transaction can be completed as usual - utilizing any of the pre-configured payment methods
-options that have been set up on the website.
+With all necessary information entered, the visitor can then click the :guilabel:`Go to Payment`
+button. Doing so first takes the visitor to a *Billing* confirmation page, followed by a *Payment*
+confirmation page, where they can utilze any configured payment method set up in the database to
+complete the order.
```

Comment:
I think it is safe to use guilabels here as well since the user is proceeding through the purchasing flow. Also fix typo :)
```suggestion
button. Doing so first takes the visitor to a :guilabel:`Billing` confirmation page, followed by a
:guilabel:`Payment` confirmation page, where they can utilize any configured payment method set up
in the database to complete the order.
```